README.md

```diff
@@ -35,7 +35,7 @@ let bitcoind = bitcoind::BitcoinD::new(bitcoind::downloaded_exe_path().unwrap())
 
 ## MSRV
 
-1.41.1
+1.53
 
```

Comment:
could you add that bitcoind `0.27.x` are the latest versions supporting MSRV `1.41.1`